src/nvim/normal.c

```diff
@@ -1340,7 +1340,7 @@ static int normal_check(VimState *state)
   quit_more = false;
 
   // If skip redraw is set (for ":" in wait_return()), don't redraw now.
-  // If there is nothing in the stuff_buffer or do_redraw is TRUE,
+  // If there is nothing in the stuff_buffer or do_redraw is true,
   // update cursor and redraw.
   if (skip_redraw || exmode_active) {
```

Comment:
@bfredl, this one is interesting, `exmode_active` is actually an int, that could be refactor as an enum.